### ISSUE TITLE
API Proposal: Archive Resolution Parameter

### ISSUE THREAD

**Issue by maikthomas** [opentok/Opentok-Python-SDK#125]
This is a request for comments regarding the API for recently added platform functionality. 
The proposal has been implemented in #124 for review.

## Archive Resolution
The platform now allows the resolution to be set for Composed archives. See [docs](https://tokbox.com/developer/rest/#start_archive)
The parameter is a string: `resolution` -  The resolution of the archive (either "640x480" or "1280x720"). This property is only set for composed archives.

We propose to add a new optional parameter to the `Opentok.start_archive()` method to enable to user to set this value.

### `start_archive()`
```python
# Backwards compatible, the parameter is optional and the default of "640x480" for composed is set by the opentok backend if no value supplied.
start_archive(sessionId)

# Default mode is Composed so this is possible.
start_archive(sessionId, resolution="1280x720")

#  Usage for SD, if omitted will be default in current Opentok API implementation anyway.
start_archive(sessionId, resolution="640x480")

# Setting in individual mode is not allowed and will throw OpentokException
start_archive(sessionId, output_mode=OutputModes.individual, resolution="640x480")

```

**Comment by maikthomas** [comment]
@jpujol I see your point, however `ArchiveResolution.640x480` is not valid syntax. I'm not sure if there's a workaround for this.
And I took `SD` and `HD` from the docs:
> The resolution of the archive, either "640x480" (SD, the default) or "1280x720" (HD). 

If we really don't want 'HD' and 'SD' I guess we can either:
- Look for alternative names for Enum
- Not use Enum, either entering values directly as "640x480", or some new pattern alltogether.

**Comment by msach22** [comment]
If that's the case, I would personally prefer using the direct value (string).

**Comment by jpujol** [comment]
@Nitrillo mentioned somewhere that he prefers `width` and `height`, and the SDK would reassemble the final string for the REST API. Not strong opinion here
Original we chose strings because it was easier to restrict the set of resolutions that we wanted
Also, is `640x480` not valid because it starts with a number? how about `r640x480` or something?

**Comment by maikthomas** [comment]
@jpujol Many thanks for the feedback.
`r640x480` would work, I tried it but aesthetically it doesn't look good and I feel we end up adding rather than removing complexity.

I also considered using standard names from here: https://en.wikipedia.org/wiki/Graphics_display_resolution
but there are some resolutions which aren't covered.

Therefore I have updated the proposal following @msach22 's suggestion to **just use the string**, this matches node and php sdks. 

I think a user could easily add `HD` and `SD` or similar as vars or enums if they so wish and map to these string values.

I have not implemented these changes in the PR yet. Will do so in the next couple days if no one objects.
